packages/automerge-repo-network-websocket/src/BrowserWebSocketClientAdapter.ts

```diff
@@ -55,24 +56,36 @@ export class BrowserWebSocketClientAdapter extends WebSocketNetworkAdapter {
     )
   }
 
+  private transmit(message: FromClientMessage) {
+    if (!this.socket || this.socket.readyState !== WebSocket.OPEN) {
+      throw new Error("Websocket Socket not ready!")
+    }
+
+    const encoded = CBOR.encode(message)
```

Comment:
alex has pointed out the CBOR encoding is a pain as a wireformat, particularly because CBOR-X is adding extra junk to the format

Reply:
Do we have a preference here? I just used the existing encoding for now.

Reply:
Obviously the versioning we have introduced here is only useful for changes outside of this encoding decision. We won't be able to change this without breaking.